src/storage/redis_metadata.h

```diff
@@ -336,3 +338,27 @@ class HyperLogLogMetadata : public Metadata {
 
   EncodeType encode_type = EncodeType::DENSE;
 };
+
+class TDigestMetadata : public Metadata {
+ public:
+  uint64_t compression;
+  uint64_t capacity;
+  uint64_t unmerged_nodes = 0;
+  uint64_t merged_nodes = 0;
+  uint64_t total_weight = 0;
+  uint64_t merged_weight = 0;
+  double minimum = std::numeric_limits<double>::infinity();
+  double maximum = -1 * std::numeric_limits<double>::infinity();
```

Comment:
I'm afraid init double via this way is not a good way, what about just min max?

Reply:
Hi @mapleFU ,

Got it! 😊

I have updated with `max` and `lowest`.

Best Regards,
Edward